nautiluszim/scraper.py

```diff
@@ -365,9 +321,17 @@ def update_metadata(self):
         self.secondary_color = self.secondary_color or secondary_color
 
         # get about content from param, archive or defaults to desc
-        self.about_content = (
-            f"<p>{self.description}</p><br><p>{self.long_description}</p>"
-        )
+
+        #setting the about_content to long_description if it is provided by the user
+        if(self.long_description is not None):
```

Comment:
Here, the about will display long_description if provided by user or just description if long_description is not provides by the user.